index.html

```diff
@@ -50,61 +50,64 @@
+            </div>    
         </div>
     </section>
-    <div class="works">
+    <div class="works" id="works_section">
         <div class="wrapper">
             <div class="mainWorks">
                 <h1>My Recent Works</h1>
                 <div class="workslineTop"></div>
-                <div class="worksCard">
+                <div class="worksCard worksCard_1">
                     <img src="images/works1.png" alt="">
-                    <span>Multi-Post Stories</span>
-                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
-                    <div class="worksLanguages">
-                        <ul class="languagesItems">
-                            <li>
-                                <a href="https://developer.mozilla.org/es/docs/Web/CSS">CSS</a>
-                            </li>
-                            <li>
-                                <a href="https://developer.mozilla.org/es/docs/Web/HTML">HTML</a>
-                            </li>
-                            <li>
-                                <a href="https://getbootstrap.com/">Bootstrap</a>
-                            </li>
-                            <li>
-                                <a href="https://www.ruby-lang.org/es/">Ruby</a>
-                            </li>
-                        </ul>
-                    </div>
-                    <a class="worksBtnSee" href="#">See Project</a>
-                </div>
-                <div class="worksCard worksCard_2">
                     <div class="rightBlock">
+                        <span>Multi-Post Stories</span>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                        <div class="worksLanguages">
+                            <ul class="languagesItems">
+                                <li>
+                                    <a href="https://developer.mozilla.org/es/docs/Web/CSS">CSS</a>
+                                </li>
+                                <li>
+                                    <a href="https://developer.mozilla.org/es/docs/Web/HTML">HTML</a>
+                                </li>
+                                <li>
+                                    <a href="https://getbootstrap.com/">Bootstrap</a>
+                                </li>
+                                <li>
+                                    <a href="https://www.ruby-lang.org/es/">Ruby</a>
+                                </li>
+                            </ul>
+                        </div>
+                        <a class="worksBtnSee" href="#">See Project</a>
+                    </div>    
+                </div>
```

Comment:
- [ ] There's an horizontal scroll bar at the bottom of your page. Kindly look into that and fix.
- [ ] Also, it is required to produce a close replica of the Figma design. This includes margin, padding among others. Kindly look into the two screenshots below and walk your design alongside the figma design.

### Your Design
![your-project-list](https://github.com/ivangonzalez224/Portfolio-mobile-first/assets/26597432/29a157df-25d6-4278-bf17-13a9f568f4d3)

### Figma Design
![figma-project-list](https://github.com/ivangonzalez224/Portfolio-mobile-first/assets/26597432/f892ee31-1e78-4fb2-b488-c98dd8648615)